src/category_theory/epi_mono.lean

```diff
@@ -48,5 +52,47 @@ lemma faithful_reflects_mono (F : C ⥤ D) [faithful F] {X Y : C} {f : X ⟶ Y}
   (hf : mono (F.map f)) : mono f :=
 ⟨λ Z g h H, F.injectivity $
   by rw [←cancel_mono (F.map f), ←F.map_comp, ←F.map_comp, H]⟩
+end
+
+class split_mono {X Y : C} (f : X ⟶ Y) :=
```

Comment:
Could you please add a docstring? Probably you should mention relation to `mono`/`epi` and `is_iso`.

Reply:
Done!